### PR TITLE
Fm/project start

### PR DESCRIPTION
**Pull request by FakharMehdii**
Created the Static design

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
+  </head>
```
Comment (muhammad-qadeer) [inline public/index.html:28]: rename this Title

Comment (JavedJutt) [inline public/index.html]: use proper title

File: src/Actions/index.js
Comment (JavedJutt) [inline src/Actions/index.js:1]: start folder names with small letter except for react components

File: src/App.test.js
Comment (JavedJutt) [inline src/App.test.js:1]: remove this file

File: src/Components/InputField/index.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+export function InputField ()
+{
+  return(
+    <input 
+    type='text'
+    id='task'
+    className='input'
```
Comment (JavedJutt) [inline src/Components/InputField/index.js]: for what purpose you have given id?

File: src/App.css [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+}
+
+.area {
+  background-color: white;
+  width: 50.8%;
+  align-items: center;
+  text-align: center;
+  padding-bottom: 20px;
+}
+
```
Comment (JavedJutt) [inline src/App.css]: create css. files with each component and import styling from there and follow conventions like inputStyle for classesNames or todoHeadingStyle etc

File: src/Components/ToDoList/index.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import { InputField } from "../InputField"
+import { ToDoTask } from "../ToDoTask"
+
+export function ToDoList ()
+{
+    return(
+        <div className='area'>
+            <h2 className='heading'>
+              My Todo        </h2>
```
Comment (JavedJutt) [inline src/Components/ToDoList/index.js]: use proper classesName

File: src/Components/ToDoTask/index.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
+export function ToDoTask () {
+  return(
+    <div className='todoTask'>
+      <div>
+        <input   className="cbox" type='checkbox'  /> 
+        <span className= 'normal'>task</span> 
+      </div>               
+      <div>
```
Comment (JavedJutt) [inline src/Components/ToDoTask/index.js]: again classesName

File: src/index.js [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+import reportWebVitals from './reportWebVitals';
+import { Provider } from 'react-redux';
+
+import { createStore } from 'redux'
+import { render } from '@testing-library/react';
+import rootReducer from './Reducer/index'
+
+
+const store = createStore(rootReducer)
+
```
Comment (JavedJutt) [inline src/index.js]: you need to create a store in different file and needs to import it here

File: src/store/index.js
Comment (JavedJutt) [inline src/store/index.js:1]: here you should create store

**Comment by JavedJutt** [comment]
left comments, pls resolve on priority

**Pull request merged**